src/Functions/keyvaluepair/impl/KeyValuePairExtractorBuilder.h

```diff
@@ -48,6 +50,7 @@ class KeyValuePairExtractorBuilder
     char quoting_character = '"';
     std::vector<char> item_delimiters = {' ', ',', ';'};
     uint64_t max_number_of_pairs = std::numeric_limits<uint64_t>::max();
+    extractKV::Configuration::UnexpectedQuotingCharacterStrategy unexpected_quoting_character_strategy = extractKV::Configuration::UnexpectedQuotingCharacterStrategy::INVALID;
```

Comment:
Am I right that this `INVALID` default is the only reason for backward incompatibility of the PR?
And if we keep it `PROMOTE` it could be `improvement`?

Reply:
Yes, you are correct. It's a choice we gotta make. IMO, `INVALID` is what should be the default, but unfortunately it is not the behavior that was implemented previously